roles/pulp-content/defaults/main.yml

```diff
@@ -1,3 +1,2 @@
 ---
-pulp_content_host: 'localhost:24816'
-pulp_content_bind: '0.0.0.0:24816'
+pulp_content_bind: '127.0.0.1:24816'
```

Comment:
The `pulp` role uses this variable before pulp-content has a chance to set the default. This var default needs to be set in `pulp/defaults/main.yml`

The primary docs should be in pulp/README.md, but the pulp-content/README.md can mention this var under `shared_variables`

Reply:
The pulp role doesn't use bind, I think only host. Also I'm going to make a followup PR (hopefully now) that will introduce another variable. I'll have that properly use the shared variable like you're saying here. Can I fix it in that PR?